packages/react-components/react-migration-v8-v9/package.json

```diff
@@ -35,6 +35,7 @@
     "@fluentui/fluent2-theme": "^8.107.51",
     "@fluentui/react": "^8.112.9",
     "@fluentui/react-components": "^9.42.0",
+    "@fluentui/react-hooks": "*",
```

Comment:
We want to take a specific caret version as our build manages bumping versions as individual packages change.

Reply:
Thanks, already fix in new commit